### PR TITLE
✨ Add jCenter integration

### PR DESCRIPTION
**Pull request by davidbilik**

### CODE REVIEW

File: android/lib.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
+VERSION_NAME=1.0.0
+VERSION_CODE=1
+GROUP=com.ackee.flipper
```
Comment (mottl-jan) [inline android/lib.properties:3]: `com`? Do we have `ackee` domain under `com`?

Comment (davidbilik) [reply android/lib.properties:3]: I don't think that it needs to match any real domain. But we are not really consistent in this across our repositories, some (and I think most of them) uses `cz` and some uses `com` .. I think in those where we expect some publicity we use `com` and in those for our purposes we have `cz`. Do you have any strong opinion for that?

Comment (mottl-jan) [reply android/lib.properties:3]: I think you should always use the domain you own. Isn't that the purpose of this convention after all? :D When somebody registers `ackee.com`, he can then create `com.ackee.flipper`. Of course it is very unlikely, but still, I think this is why this convention exists. Maybe you can ask someone else on the opinion about this.

Comment (davidbilik) [reply android/lib.properties]: Ok, those conventions exists for some reason :D

**Pull request merged**